src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin arrow-animation-properties {
```

Comment:
The mixin name should follow BEM naming conventions. Consider renaming it to 'mixin-arrow-animation-properties' or something similar.